pom.xml

```diff
@@ -148,6 +148,12 @@
             <version>5.3.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.sun.net.httpserver</groupId>
+            <artifactId>http</artifactId>
+            <version>20070405</version>
```

Comment:
@sushantdewan123 Are we clear to depend on this component (licensing-wise)? This looks like it was part of Java SE 6.

Reply:
I am happy to change this out. @basilisk487, any suggestions for a replacement? It just needs to allow connections in tests.